pkg/kubelet/kubelet_pods_linux_test.go

```diff
@@ -39,6 +39,10 @@ func TestMakeMounts(t *testing.T) {
 	propagationBidirectional := v1.MountPropagationBidirectional
 	propagationNone := v1.MountPropagationNone
 
+	testKubelet := newTestKubelet(t, false /* controllerAttachDetachEnabled */)
+	defer testKubelet.Cleanup()
+	kubelet := testKubelet.kubelet
```

Comment:
personally I'd prefer just do `testKubelet.kubelet.makeMounts` below, any reason to do it this way?